### PR TITLE
Feat/import path

### PR DESCRIPTION
**Pull request by SallesCosta**
Adicionada regra sobre `absolute paths` ou `relative paths`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +105,12 @@ if(!isWholeCodeCoveraged) return
 - Keep the current tests and make them pass! 
 Once the current code is tested and can be refactored. We must make sure that the new changes will not break the current tests. 
 
+### 1.3 Imoprts
```
Comment (felipefialho) [inline README.md:108]: (block) `imports`

**Review by felipefialho — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SallesCosta) [reply README.md]: [OK](https://github.com/juntossomosmais/frontend-guideline/pull/33/commits/508fda3bc3abda3263701e71ecc518347852b926)

File: README.md [hunk 2]
```diff
@@ -104,6 +105,12 @@ if(!isWholeCodeCoveraged) return
 - Keep the current tests and make them pass! 
 Once the current code is tested and can be refactored. We must make sure that the new changes will not break the current tests. 
 
+### 1.3 Imports
+
+If the data to be imported belongs to `another module/scope`, use an **absolute path**.
+
+If the data to be imported belongs to the `same module/scope`, use **relative path**.
```
Comment (mauriciomutte) [inline README.md:112]: You could add some example. What do you think?

**Review by mauriciomutte — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -104,6 +105,12 @@ if(!isWholeCodeCoveraged) return
 - Keep the current tests and make them pass! 
 Once the current code is tested and can be refactored. We must make sure that the new changes will not break the current tests. 
 
+### 1.3 Imports
+
+If the data to be imported belongs to `another module/scope`, use an **absolute path**.
```
Comment (felipefialho) [inline README.md:110]: (fix) 

```suggestion
If the data to be imported belongs to **another module/scope**, use an **absolute path**.
```

File: README.md (hunk 2 above)
Comment (felipefialho) [inline README.md:112]: (fix)
```suggestion
If the data to be imported belongs to the **same module/scope**, use **relative path**.
```

Comment (SallesCosta) [reply README.md]: [Done](https://github.com/juntossomosmais/frontend-guideline/pull/33/commits/331f5c371a789fdef18f3668a72c642551df1040)

**Review by felipefialho — APPROVED** [review]
🚀

**Review by mauriciomutte — APPROVED** [review]

**Pull request closed without merging**